css/Components/Text.scss

```diff
@@ -12,5 +13,16 @@ $Text-font-weight: normal;
   line-height: $Text-line-height;
   font-family: $Text-font-family;
   font-weight: $Text-font-weight;
-  
+}
+
+.Text.Text--white {
```

Comment:
I can understand the uppercase modifier, but if you're changing the color and the size this is no longer the `Text` component. I would suggest for you to move this to the `Link`